pkg/controllers/gitrepository/git_repository.go

```diff
@@ -17,3 +19,30 @@ type GiteaClient interface {
 	SetBasicAuth(username, password string)
 	SetContext(ctx context.Context)
 }
+
+type gitHubClientI interface {
```

Comment:
```suggestion
type GitHubClient interface {
```

to go with the GiteaClient or we should change the other one to `giteaClient`. Also I dont think we need the `I` for the interface name. not a common practice in this code base.

Reply:
yeah not sure how it got in there. I think I was messing with it when I was refactoring.